packages/synapse-react-client/src/components/SetBasicAccessRequirementFields/GovernanceUtils.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE,
+  AccessRequirement,
+  LOCK_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE,
+  MANAGED_ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE,
+  ManagedACTAccessRequirement,
+  SELF_SIGN_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE,
+  SelfSignAccessRequirement,
+  TERMS_OF_USE_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE,
+} from '@sage-bionetworks/synapse-types'
+
+export const LOCK_ACCESS_REQUIREMENT_TEXT =
+  'Access restricted pending review by Synapse Access and Compliance Team.'
+
+export const getOldAccessRequirementInstructions = (ar: AccessRequirement) => {
```

Comment:
Replicates this [SWC method](https://github.com/Sage-Bionetworks/SynapseWebClient/blob/9085d9d2f9644ba7ceb9db9624b3a48e52a269a4/src/main/java/org/sagebionetworks/web/client/utils/GovernanceServiceHelper.java#L40), other than handling [PostMessageContentAccessRequirement](https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/PostMessageContentAccessRequirement.html), which is deprecated and shouldn't be used.